.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -18,6 +18,16 @@ can uncomment the below line to indicate which issue your PR fixes, for example
 Fixes #
 -->
 
+### Testing done
+<!--
+How has this code been tested?
+
+Before reviewers can be confident in the correctness of a changeset,
```

Comment:
"Changeset" is an unusual term for Github interactions. I suggest "pull request" would be more idiomatic.